### ISSUE TITLE
TrustKit 1.5.1 build issues with mopub-ios-sdk 4.16.0

### ISSUE THREAD

**Issue by jtrespalacios** [datatheorem/TrustKit#148]
In attempting to integrate TrustKit into a project I've encountered a compilation error as a result in the mopub-ios-sdk.

```
.../MoPubSDK/Internal/Event Logging/MPQRunLoopOperation.m:117:9: error: implicit declaration of function 'assert' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
        assert(newState > _state);
        ^
1 error generated.
```

The complete build log is attached.  I've tried only including the two pods in a project and am able to reproduce this.  When only including on library or the other the project builds successfully.

Attached is a build log when attempting to use both.

[mopub trustkit build log](https://github.com/datatheorem/TrustKit/files/1408318/mopub.trustkit.build.log)

**Comment by nabla-c0d3** [comment]
Hello,
Thanks for the report - we'll take a look. Is the SDK publicly available?
Thanks!

**Comment by nabla-c0d3** [comment]
It looks it might just be TrustKit's build settings that are too strict (especially `-Werror`) for building the mopub SDK. However I am not sure why TrustKit's settings would be used when building mopub?

**Comment by nabla-c0d3** [comment]
Closed as a duplicate of #150
